db/migrate/20220110190210_add_prod_configto_sp_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddProdConfigtoSpTable < ActiveRecord::Migration[6.1]
+  def change
+    add_column :service_providers, :prod_config, :boolean
```

Comment:
**suggestion:** do we want to set a default (presumably `false`) and also prevent null values?

Reply:
+1 on this comment

Reply:
Makes sense to me.